cmake/get_boost.cmake

```diff
@@ -0,0 +1,74 @@
+set(BOOST_REQUESTED_VERSION 1.69.0 CACHE STRING "")
+set(BOOST_SHA1 8f32d4617390d1c2d16f26a27ab60d97807b35440d45891fa340fc2648b04406 CACHE STRING "")
```

Comment:
Why can't you get boost from apt-get ?

Reply:
Ideally that would be the case. But 16.04 is on 1.58 and we need a newer one. We are using the library `Boost.Beast` which was introduced in 1.66 and has improved in performance + usability since then.